lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -192,7 +215,7 @@ def _get_server_cert(server_cert_getter, self, charm):
         return
     elif not isinstance(server_cert, str):
```

Comment:
should probably take `str | Path` and make it clear we're assuming it's absolute

Reply:
True, took that into account and added a check if the path is absolute.